subs2srs.lua

```diff
@@ -399,9 +406,16 @@ end
 
 local function update_last_note(overwrite)
     maybe_reload_config()
-    local sub = subs_observer.collect(n_lines)
+    local sub
+    local n_lines = quick_creation_opts:get_lines()
+    local n_cards = quick_creation_opts:get_cards()
+    if n_lines then
+      sub = subs_observer.collect_from_all_dialogues(n_lines)
+    else
+      sub = subs_observer.collect_from_current()
+    end
     -- this now returns a table
-    local last_note_ids = ankiconnect.get_last_note_ids(n_cards or 1)
+    local last_note_ids = ankiconnect.get_last_note_ids(n_cards)
```

Comment:
What's the purpose of the `or 1` expression?

Reply:
I don't quite remember why I had that to be honest lol. Likely some sort of safeguard in case n_cards is nil or 0, but using the quick_creation_opts guarantees a valid value now.